lua/core/lualine/utils.lua

```diff
@@ -5,7 +5,7 @@ function M.validate_theme()
 
   local lualine_loader = require "lualine.utils.loader"
   local ok = pcall(lualine_loader.load_theme, theme)
-  if not ok then
+  if not ok and type(theme) ~= "table" then
```

Comment:
You ended up having a case for this ?

Reply:
Yes.
lualine accepts two types of theme, ```string``` or ```table```.
A custom theme would be table type and the second condition is added
to prevent custom theme from falling back to ```auto``` theme.

Reply:
I'm not very happy that the entry can accept two different types, but let's keep it like this for now.
Maybe, to make it clearer, you could do smth like this:

``` lua
  local theme = lvim.builtin.lualine.options.theme
  if type(theme) == "table" then
    return -- user specified theme
  end

  local lualine_loader = require "lualine.utils.loader"
  local ok = pcall(lualine_loader.load_theme, theme)
  if not ok then
    lvim.builtin.lualine.options.theme = "auto"
  end

```